scripts/workers/code_upload_submission_worker.py

```diff
@@ -567,6 +589,19 @@ def update_failed_jobs_and_send_logs(
                             logger.exception(
                                 "Exception while reading Job logs {}".format(e)
                             )
+                    elif is_submission_container_executed(
+                        container_name, container_state, container_state_map
+                    ):
+                        cleanup_submission(
```

Comment:
@savish28 Refactoring: This call is repeated in this function 3 times with only one param being different. Just use a boolean to set cleanup and store the log/error. Then just call the function once at the end

Reply:
Good idea,sure.

Reply:
Updated.